src/test/java/api/support/fixtures/ManualBlocksFixture.java

```diff
@@ -0,0 +1,39 @@
+package api.support.fixtures;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+
+import java.net.MalformedURLException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import api.support.builders.ManualBlockBuilder;
+import api.support.http.ResourceClient;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+
+public class ManualBlocksFixture {
+  private final RecordCreator manualBlocksRecordCreator;
+
+  public ManualBlocksFixture(ResourceClient manualBlocksClient) {
+    manualBlocksRecordCreator = new RecordCreator(manualBlocksClient,
+      materialType -> getProperty(materialType, "id"));
```

Comment:
Why is material type relevant for manual blocks?

Reply:
Mistype. Renamed